<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-27T19:35:52 ECT" name="Default suite" finished-at="2023-10-27T19:35:54 ECT" duration-ms="1992">
    <groups>
    </groups>
    <test started-at="2023-10-27T19:35:52 ECT" name="Default test" finished-at="2023-10-27T19:35:54 ECT" duration-ms="1992">
      <class name="ec.com.learning.restassured.restfulbooker.GetBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:ec.com.learning.restassured.restfulbooker.GetBookingTests@59d016c9]" started-at="2023-10-27T19:35:52 ECT" name="setUp" finished-at="2023-10-27T19:35:52 ECT" duration-ms="330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getBookingTest()[pri:0, instance:ec.com.learning.restassured.restfulbooker.GetBookingTests@59d016c9]" started-at="2023-10-27T19:35:52 ECT" name="getBookingTest" finished-at="2023-10-27T19:35:54 ECT" duration-ms="1633" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
firstname in response is not expected expected [Mary] but found [Susan],
lastname in response is not expected expected [Jackson] but found [Smith],
totalprice in response is not expected expected [246] but found [136],
checkin in response is not expected expected [2015-05-19] but found [2015-05-05],
checkout in response is not expected expected [2023-04-09] but found [2018-10-22],
additionalneeds in response is not expected expected [Breakfast] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
firstname in response is not expected expected [Mary] but found [Susan],
lastname in response is not expected expected [Jackson] but found [Smith],
totalprice in response is not expected expected [246] but found [136],
checkin in response is not expected expected [2015-05-19] but found [2015-05-05],
checkout in response is not expected expected [2023-04-09] but found [2018-10-22],
additionalneeds in response is not expected expected [Breakfast] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at ec.com.learning.restassured.restfulbooker.GetBookingTests.getBookingTest(GetBookingTests.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getBookingTest -->
      </class> <!-- ec.com.learning.restassured.restfulbooker.GetBookingTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
